### PR TITLE
feat(xrpl_cli): introduce offers create command

### PR DESCRIPTION
**Pull request by gmosx**

### CODE REVIEW

**Comment by gmosx** [comment]
cc @allanbrondumkr

File: xrpl_types/src/amount.rs [hunk 1]
```diff
@@ -40,6 +40,38 @@ impl Amount {
         }
     }
 
+    /// Try to construct an amount from a 'spec' string.
+    /// Examples:
+    /// Amount::try_from_str("26.231 USD:rhub8VRN55s94qWKDv6jmDy1pUykJzF3wq");
```
Comment (allanbrondumkr) [inline xrpl_types/src/amount.rs:45]: It this an "official" format or a format specific to the cli application. If it is the latter, then the logic should be in the `xrpl_cli` crate?

Comment (allanbrondumkr) [reply xrpl_types/src/amount.rs:45]: At least I have not seen it used in the XRPL docs?

**Comment by gmosx** [comment]
Yeah, I had the same thoughts, I even added a comment in a follow up PR. I will probably move this out.

**Pull request merged**